about.html

```diff
@@ -37,7 +37,7 @@
           <a href="">&times;</a>
           <a href="index.html" class="close-menu">Home</a>
           <a href="about.html" class="close-menu">About Us</a>
-          <a href="#" class="close-menu">Inquirires</a>
+          <a href="" class="close-menu">Inquirires</a>
           <a href="#" class="close-menu">Partner</a>
        </div>
```

Comment:
- [ ] The mobile menu should be the same on the Home page and About page, kindly consider fixing this so the menu is the same across the website 

|The Home page|The About page|
|-|-|
|![Screenshot_20230706_200825](https://github.com/paulshaun-kdtk/capstone-Car-Show/assets/25227022/419a586b-fd76-4261-9087-0cdc6fbdd746)|![Screenshot_20230706_200839](https://github.com/paulshaun-kdtk/capstone-Car-Show/assets/25227022/19b1d7ef-8aea-41a3-b327-ebd222b64b66)|